packages/cli-lib/templates.js

```diff
@@ -27,6 +31,16 @@ const getFileAnnotations = filePath => {
   }
 };
 
+const getAnnotationsFromSource = source => {
```

Comment:
This looks very similar to `getFileAnnotations`. Could you update `getFileAnnotations` to use this new `getAnnotationsFromSource` func? Just to DRY things up a little.

Reply:
Agreed, maybe `getFileAnnotations` can use `getAnnotationsFromSource`?

Reply:
Ah, good call. Will update